parts/linux/cloud-init/artifacts/README-COMPONENTS.md

```diff
@@ -131,7 +132,8 @@ Here are the explanation of the above schema.
 1. In `downloadURIs`, we can define different OS distro. For now for Linux, we have _ubuntu_, _mariner_, _marinerkata_ and _default_.
 1. There are 3 types of OSDistro
     - In `UbuntuOSDistro`, we can define different OS release versions. For example, `r1804` implies release 18.04.
-    - In `MarinerOSDistro`, we only have `current` now, which implies that single configurations will be applied to all Mariner release versions. We can distinguish them in needed.
+     - In `MarinerOSDistro`, we only have `current` now, which implies that single configurations will be applied to all Mariner release versions. We can distinguish them in needed. Note, we confirmed with Mariner team, Azure Linux 2.0 is reporting itself as `mariner` in the file `/etc/os-release`. So for Azure Linux 2.0 case, it will still read the package versions from `mariner` block.
```

Comment:
Does current actually apply to anything at this point?

Reply:
No it doesn't. Since AzureLinux V2 reports itself as Mariner, it will use the mariner config. So Azurelinux.current isn't applying to anything now.
Good point, I will update the readme to describe this. Will also delete the Azurelinux.current from components.json